Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/SingleFileTransfer.cs

```diff
@@ -273,20 +273,18 @@ private void RestoreModified()
 
     private static Encoding GetEncoding(Destination dest)
```

Comment:
The same comment as in the other PR about making this method and the GetEncoding in FileTransporter use one method instead of two.

Reply:
Fixed